ckb/src/main/java/org/nervos/ckb/type/param/OutputsValidator.java

```diff
@@ -0,0 +1,17 @@
+package org.nervos.ckb.type.param;
+
+/** Copyright © 2020 Nervos Foundation. All rights reserved. */
+public enum OutputsValidator {
+  DEFAULT("default"),
+  PASSTHROUGH("passthrough");
+
+  private final String value;
+
+  OutputsValidator(String value) {
+    this.value = value;
```

Comment:
Can I set a value that is neither "default" nor "passthrough"?

Reply:
No. Because `OutputsValidator(String value)` is private to outside caller and it is just used inner assignment.